src/lib/UniversalDiscovery/Event/Subscriber/ReadAllowedContentTypes.php

```diff
@@ -83,9 +83,7 @@ public function onUdwConfigResolve(ConfigResolveEvent $event): void
             return;
         }
 
-        if ($this->allowedContentTypesIdentifiers === null) {
-            $this->allowedContentTypesIdentifiers = $this->getAllowedContentTypesIdentifiers($config['allowed_content_types'] ?? []);
-        }
+        $this->allowedContentTypesIdentifiers = $this->getAllowedContentTypesIdentifiers($config['allowed_content_types'] ?? []);
```

Comment:
To increase readability.
```suggestion
        $this->allowedContentTypesIdentifiers = $this->getAllowedContentTypesIdentifiers(
            $config['allowed_content_types'] ?? []
        );
```